pkg/features/pilot/pilot.go

```diff
@@ -126,6 +126,13 @@ var (
 	// for cache sync before Pilot bootstrap. Set env PILOT_ENABLE_WAIT_CACHE_SYNC = 0 to disable it.
 	EnableWaitCacheSync = env.RegisterStringVar("PILOT_ENABLE_WAIT_CACHE_SYNC", "", "").Get() != "0"
 
+	enableFallthroughRouteVar = env.RegisterBoolVar(
+		"PILOT_ENABLE_FALLTHROUGH_ROUTE",
+		false,
+		"EnableFallthroughRoute provides an option to add a final wildcard match for routes. When ALLOW_ANY traffic policy is used, a Passthrough cluster is used. When REGISTRY_ONLY traffic policy is used, a 502 error is returned.",
```

Comment:
@geeknoid is this how the env framework is intended to be used? I noticed all the other ones have comments instead of docs so wanted to verify.